pkg/acsengine/k8s_versions.go

```diff
@@ -36,58 +36,60 @@ var k8sComponentVersions = map[string]map[string]string{
 		"gclowthreshold":  strconv.Itoa(DefaultKubernetesGCLowThreshold),
 	},
 	"1.10": {
-		"dockerEngine":    "1.13.*",
-		"dashboard":       "kubernetes-dashboard-amd64:v1.8.3",
-		"exechealthz":     "exechealthz-amd64:1.2",
-		"addon-resizer":   "addon-resizer:1.8.1",
-		"heapster":        "heapster-amd64:v1.5.1",
-		"metrics-server":  "metrics-server-amd64:v0.2.1",
-		"kube-dns":        "k8s-dns-kube-dns-amd64:1.14.8",
-		"addon-manager":   "kube-addon-manager-amd64:v8.6",
-		"dnsmasq":         "k8s-dns-dnsmasq-nanny-amd64:1.14.8",
-		"pause":           "pause-amd64:3.1",
-		"tiller":          "tiller:v2.8.1",
-		"rescheduler":     "rescheduler:v0.3.1",
-		"aci-connector":   "virtual-kubelet:latest",
-		"nodestatusfreq":  DefaultKubernetesNodeStatusUpdateFrequency,
-		"nodegraceperiod": DefaultKubernetesCtrlMgrNodeMonitorGracePeriod,
-		"podeviction":     DefaultKubernetesCtrlMgrPodEvictionTimeout,
-		"routeperiod":     DefaultKubernetesCtrlMgrRouteReconciliationPeriod,
-		"backoffretries":  strconv.Itoa(DefaultKubernetesCloudProviderBackoffRetries),
-		"backoffjitter":   strconv.FormatFloat(DefaultKubernetesCloudProviderBackoffJitter, 'f', -1, 64),
-		"backoffduration": strconv.Itoa(DefaultKubernetesCloudProviderBackoffDuration),
-		"backoffexponent": strconv.FormatFloat(DefaultKubernetesCloudProviderBackoffExponent, 'f', -1, 64),
-		"ratelimitqps":    strconv.FormatFloat(DefaultKubernetesCloudProviderRateLimitQPS, 'f', -1, 64),
-		"ratelimitbucket": strconv.Itoa(DefaultKubernetesCloudProviderRateLimitBucket),
-		"gchighthreshold": strconv.Itoa(DefaultKubernetesGCHighThreshold),
-		"gclowthreshold":  strconv.Itoa(DefaultKubernetesGCLowThreshold),
+		"dockerEngine":       "1.13.*",
+		"dashboard":          "kubernetes-dashboard-amd64:v1.8.3",
+		"exechealthz":        "exechealthz-amd64:1.2",
+		"addon-resizer":      "addon-resizer:1.7",
```

Comment:
I assume we're downgrading addon-resizer to 1.7 because 1.8.1 is not mutually compatible with cluster-autoscaler?

Can we be confident that we won't re-introduce this bug:

https://github.com/Azure/acs-engine/issues/2430

Which was fixed here:

https://github.com/Azure/acs-engine/pull/2753

(the addon-resizer manfest definition was changed with the above PR, not just the version; perhaps the changes in how addon-resizer was expressed is the important part and not the upgrade to 1.8.1?)

Reply:
didn't mean to downgrade addon-resizer, probably didn't notice when rebasing. reverted that change.